### DISCUSSION TITLE
Jolokia endpoint doesnt work for Spring 3.x applications

### DISCUSSION THREAD

**Discussion by parthiuk** [rhuss/jolokia#777] [General]
Hi,

I use jolokia to read the metrics from my SpringBoot 2.x applications.  "/jolokia" works fine to read all the metrics.

When I try to upgrade my application for Spring Boot 3.x. "/jolokia" end point is not responding and throws Error 400.

Even tried to use Jolokia 1.7.1, still no luck.  Could you please help me to resolve this?

**Comment by tadayosi** [comment]
Hi @parthiuk,

For Spring Boot 3.x, try using Jolokia 2.x, the latest one is 2.1.2. You can find the instructions on how to use Jolokia with SB 3 in details here:
https://jolokia.org/reference/html/manual/spring.html
